service/implementation/email_sender/test.py

```diff
@@ -0,0 +1,9 @@
+from api.routes.__init__ import app
+
+def test():
```

Comment:
Don't want to see any test/commented/legacy/etc code in PRs.
Please, have it removed